### DISCUSSION TITLE
Mark image as favorite with keyboard shortcut

### DISCUSSION THREAD

**Discussion by Pryla** [photoprism/photoprism#2546] [General]
As describe [here](https://github.com/photoprism/photoprism/issues/508) a while ago there have been thoughts about adding keyboard shortcuts to this awesome project. Does anybody how far this went or where we are at the moment?

Thank you!

**Comment by graciousgrey** [comment]
Here is the related issue: https://github.com/photoprism/photoprism/issues/1131 it has not been started yet.

Our [Project Roadmap](https://link.photoprism.app/roadmap) shows what tasks are in progress and what features will be implemented next. You may give ideas you like a thumbs-up, so we know what's most popular.

Be aware that we have a zero-bug policy and do our best to help users when they need support or have other questions. This comes at a price though, as we can't give exact release dates for new features. Our team receives many more requests than can be implemented, so we want to emphasize that we are in no way obligated to implement the features, improvements, or other changes you request. We do, however, appreciate your feedback and carefully consider all requests.

Since sustained funding is key to quickly releasing new features, we encourage all users to support our mission by [signing up as a sponsor](https://photoprism.app/membership).

**Comment by tsvetkovv** [comment]
@Pryla I faced with the same need. As a workaround I just wrote a JS script. May be useful for somebody
```js
document.body.onkeyup = function(e) {
  if (e.code == "Space") {
   document.querySelector('.action-select').click()
  }
}
```
`.action-select` can be replaced with `.action-like`.
